web/admin/src/SessionBox.js

```diff
@@ -99,10 +100,21 @@ export class SessionBox extends Component {
 
     handleNewSession = () => {
       const current = this.state.showNewSession
-      this.setState({showNewSession: !current})
+      this.setState({showNewSession: !current, color: "#FAFAFA"})
     }
 
     updateList = (value) => {
+      var queryText = value.toLowerCase()
+      if (queryText.length > 0){
+        this.setState({search: true, searchValue: value})
+      }
+      else {
+        this.setState({search: false, searchValue: value})
+      }
+    }
+
+    returnNewList = () => {
+      let value = this.state.searchValue
```

Comment:
Why is this a `let`?  Also, `value` does not seem to be used beyond the line directly below it, so it could just be `const queryText = this.state.searchValue.toLowerCase()`